feflow/protocols/nonequilibrium_cycling.py

```diff
@@ -991,12 +991,21 @@ def _create(
         mapping: Optional[Union[ComponentMapping, list[ComponentMapping]]] = None,
         extends: Optional[ProtocolDAGResult] = None,
     ) -> list[ProtocolUnit]:
+        from openfe.protocols.openmm_rfe.equil_rfe_methods import (
+            _validate_alchemical_components,
+        )
+        from openfe.protocols.openmm_utils import system_validation
+
         # Handle parameters
         if mapping is None:
```

Comment:
This check is probably now redundant if we merge these changes. I would prefer the approach that I tried in #55 where each validation has its own method, just for modularity and compartmentalization.

Reply:
removed in [55b8765](https://github.com/OpenFreeEnergy/feflow/pull/90/commits/55b876519d202cd35cf2d5c738b27b6320e60c23)